fips/serializers.py

```diff
@@ -0,0 +1,7 @@
+from rest_framework import serializers
+
+
+class FipsSerializer(serializers.Serializer):
+    name = serializers.CharField(max_length=1024)
+    type = serializers.CharField(max_length=2)
+    fips_id = serializers.IntegerField()
```

Comment:
Is this necesary? i thought that without specifying anything, the Serializer would default to exposing all fields on the model.